issue_25/get_seq.py

```diff
@@ -0,0 +1,51 @@
+# -*- coding: utf-8 -*-
```

Comment:
Please include your function in the module `htsinfer/infer_read_orientation.py` and call the function in its `infer()` function. There should be an example in there already on how to do that.
You don't need to add author info as this will be tracked by Git / GitHub automatically, on a per-line basis :)